project/Build.scala

```diff
@@ -278,7 +280,9 @@ object Scrooge extends Build {
       publishMavenStyle := false,
       repository in bintray := "sbt-plugins",
       licenses += (("Apache-2.0", url("http://www.apache.org/licenses/LICENSE-2.0.html"))),
-      bintrayOrganization in bintray := Some("twittercsl")
+      bintrayOrganization in bintray := Some("twittercsl"),
+      scriptedBufferLog := false,
```

Comment:
The documentation for scripted is here: http://www.scala-sbt.org/0.13/docs/Testing-sbt-plugins.html. Do you want me to just put that link in a comment?